2010-06-04  Michael Tautschnig  <mt@debian.org>

	* install_packages: Fix typo in info message (thanks Adrian Glaubitz
		<glaubitz@physik.fu-berlin.de>)
Index: trunk/bin/install_packages
===================================================================
--- trunk.orig/bin/install_packages
+++ trunk/bin/install_packages	
@@ -112,7 +112,7 @@
 $verbose=$ENV{verbose} || $opt_v;
 $opt_d && setdownloadonly();
 $maxpl=$ENV{MAXPACKAGES} || $opt_m ;
-$maxpl && $verbose && warn "Maxmimum number of packages installed at a time set to $maxpl\n";
+$maxpl && $verbose && warn "Maximum number of packages installed at a time set to $maxpl\n";
 $maxpl or $maxpl = 99 ; # set default value
 
 my $qopt="-qq" unless $verbose;
